core-plugins/src/test/java/co/cask/hydrator/plugin/batch/joiner/JoinerConfigTest.java

```diff
@@ -59,38 +56,25 @@ public void testJoinerConfig() {
       Schema.Field.of("renamed_actor", Schema.of(Schema.Type.STRING)),
       Schema.Field.of("renamed_category", Schema.of(Schema.Type.STRING)));
 
-    Map<String, Map<String, String>> selectedFields = new HashMap<>();
-    selectedFields.put("film", com.google.common.collect.ImmutableMap.of("film_id", "film_id", "film_name",
-                                                                         "film_name"));
-    selectedFields.put("filmActor", com.google.common.collect.ImmutableMap.of("actor_name", "renamed_actor"));
-    selectedFields.put("filmCategory", com.google.common.collect.ImmutableMap.of("category_name", "renamed_category"));
-
-
     JoinerConfig config = new JoinerConfig("film.film_id=filmActor.film_id=filmCategory.film_id&" +
                                              "film.film_name=filmActor.film_name=filmCategory.film_name",
-                                           GSON.toJson(selectedFields), "film,filmActor,filmCategory");
+                                           selectedFields, "film,filmActor,filmCategory");
 
     MockPipelineConfigurer pipelineConfigurer = new MockPipelineConfigurer(
```

Comment:
shouldn't need to use MockPipelineConfigurer anywhere, just call the relevant method on the config object.